data/en/2_0001.json

```diff
@@ -6,25 +6,54 @@
+                        },
+                        {
+                            "description": "Data should relate to all sites in England, Scotland and Wales. You can provide informed estimates if actual figures aren't available."
+                        },
+                        {
+                            "description": "All information you provide is strictly confidential. "
+                        }
+                    ]
+                }],
+                "preview_content": {
+                    "id": "preview",
+                    "title": "Information you need",
                     "content": [{
                         "list": [
-                            "Please complete this statutory questionnaire for {{respondent.trad_as_or_ru_name}}, for the period {{exercise.start_date|format_date}}.",
-                            "Data should relate to all sites in England, Scotland and Wales, unless otherwise stated in the ‘trading as’ name, above."
+                            "Total number of employees",
+                            "Male employees working more than 30 hours per week",
+                            "Male employees working 30 hours or less per week",
+                            "Female employees working more than 30 hours per week",
+                            "Female employees working 30 hours or less per week"
+                        ]
+                    }, {
+                        "title": "Include:",
+                        "list": [
+                            "all workers paid directly from this business’s payroll(s)",
+                            "those temporarily absent but still being paid, for example on maternity leave"
+                        ]
+                    }, {
+                        "title": "Exclude:",
+                        "list": [
+                            "agency workers paid directly from the agency payroll",
+                            "voluntary workers",
+                            "former employees only receiving a pension",
+                            "self-employed workers",
+                            "working owners who are not paid via PAYE"
                         ]
                     }]
-                }, {
-                    "id": "information-to-provide",
-                    "type": "Basic",
-                    "title": "You will be asked to provide information for the business, including:",
+                },
+                "secondary_content": [{
+                    "title": "How we use your data",
                     "content": [{
                         "list": [
-                            "Employee numbers"
+                            "You cannot appeal your selection. Your business was selected to give us a comprehensive view of the UK economy.",
+                            "The information you provide contributes to the Labour Market Statistics."
```

Comment:
Should `Labour Market Statistics` link to https://www.ons.gov.uk/employmentandlabourmarket/peopleinwork/employmentandemployeetypes/bulletins/uklabourmarket/previousReleases

Reply:
@jonnyshaw89 Checked with @charlotte-hirst before implementing this and was agreed that it will not be a link.